roboteam_ai/src/control/ControlUtils.cpp

```diff
@@ -286,6 +304,10 @@ std::vector<std::pair<Vector2, Vector2>> ControlUtils::calculateClosestPathsFrom
     return solutionPairs;
 }
 
+Vector2 ControlUtils::computeSimpleReceivePos(Vector2 startPos, Vector2 robotPos){
```

Comment:
this is not something that belongs in controlUtils I think. Most controlUtils are about vector calculus etc.

Reply:
moved it